docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md

```diff
@@ -38,16 +38,21 @@ Using federated authentication, you can use an IdP to manage access to your Yuga
 
 Note that after federated authentication is enabled, only Admin users can sign in using email-based login.
 
-Currently only the Microsoft Entra ID IdP and the OIDC protocol are supported.
+Currently, YugabyteDB Managed supports Microsoft Entra ID<!--, PingOne,--> and Okta for Enterprise IdPs, and the OIDC protocol.
```

Comment:
Suggestion -> 
`YugabyteDB Managed currently supports Microsoft Entra ID and Okta for Enterprise Identity Providers (IdPs) exclusively through the OIDC (OpenID Connect) protocol.`
Mainly because both can be SAML apps as well which we do not currently support
And I think once we have Ping as well in this we can list the three with bullet points mentioning OIDC protocol in bracket